src/modules/views/SharedDrive/SharedDriveFolderView.jsx

```diff
@@ -43,7 +45,12 @@ const SharedDriveFolderView = () => {
         <Content className={isMobile ? '' : 'u-pt-1'}>
           <FolderViewHeader>
             <SharedDriveBreadcrumb driveId={driveId} />
-            <Toolbar canUpload={false} canCreateFolder={false} />
+            <Toolbar
+              canUpload={false}
+              canCreateFolder={false}
```

Comment:
i know this is not your change, but why canUpload and canCreateFolder are false?